### DISCUSSION TITLE
GPU support for applying SHAP on BERT models

### DISCUSSION THREAD

**Discussion by amansinha09** [shap/shap#3888] [Q&A]
Hello, 

I am running shap.explainer over text datasets, my models are from BERT family (pytorch framework). But, running the shap.explainer takes a lot of time. I wonder if there is GPU support to run mutiple datapoints in parallel some suggestions to speed up the shap calculations.

Thanks in advance.

**Comment by Ningshiqi** [comment]
I follow the [https://captum.ai/tutorials/Llama2_LLM_Attribution](url) , I use llama2-7b-hf.
In Colab T4 gpu,it always runing, didn't stop.  Is anyone came across this problem?

**Reply by Ningshiqi** [reply]
47 MIN
